### PR TITLE
Some updates for the docs

### PR DESCRIPTION
**Pull request by Senyoret1**
Changes:
- The documentation about the `devRecoveryDevice` and `devGenerateMnemonic` functions was updated to include changes made in past PRs.

- The entry in the changelog about the rng dependency was updated.

Does this change need to mentioned in CHANGELOG.md?
No

Requires testing
No

Comments about testing , should you have some
None

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -287,7 +287,8 @@ asking for it very frequently).
 *Return value:*
 
 A promise that receives a text string that depends on the result of the operation:
-- If the hardware wallet alreadys has a mnemonic: `Generate Mnemonic operation failed or refused`.
+- If the hardware wallet already has a mnemonic or `wordCount` has an invalid value
+(promise rejected): `Error: Generate Mnemonic operation failed or refused`.
```
Comment (olemis) [inline DOCUMENTATION.md]: single line

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -354,8 +355,8 @@ asking for it very frequently).
 *Return value:*
 
 A promise that receives a text string that depends on the result of the operation:
-- If the wallet already has a mnemonic or the user cancels the operation (promise rejected):
-`Error: Expected WordAck after Button confirmation`.
+- If the wallet already has a mnemonic, the user cancels the operation or `wordCount` has an invalid value
+(promise rejected): `Error: Expected WordAck after Button confirmation`.
```
Comment (olemis) [inline DOCUMENTATION.md]: single line

File: DOCUMENTATION.md (hunk 1 above)
Comment (Senyoret1) [reply DOCUMENTATION.md]: The new line is to follow the format used in the rest of the file: almost all long lines are being split into several lines to make editing a bit easier. Markdown ignores the new line and shows all as a single line.

**Comment by Senyoret1** [comment]
Integrated in https://github.com/skycoin/hardware-wallet-js/pull/83

**Pull request closed without merging**